docs/source/models/intervention_models/alzheimers/testing_diagnosis.rst

```diff
@@ -236,15 +238,16 @@ Assumptions and Limitations
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
 - Since BBBM testing eligibility is pre-clinical stage and CSF/PET is MCI or AD dementia stage, 
   and simulants cannot move backwards, CSF/PET test history is irrelevant to BBBM 
-  test eligibility  
+  test eligibility;
 - The same simulants undergo repeat testing to reflect ongoing issues with access or insurance,
-  so propensity does not need to be re-assigned at any point.
+  so propensity does not need to be re-assigned at any point;
 - Since BBBM uses the same propensity as existing testing, BBBM should replace many CSF and PET
-  tests, though some simulants may not qualify for BBBM tests due to age requirements, or may get a BBBM false negative.
+  tests, though some simulants may not qualify for BBBM tests due to age requirements, or may get a BBBM false negative;
+- The determination of whether a test is a false positive is independent for healthy individuals, and precludes the possibility that individual characteristics like protein expression levels or chronic kidney disease are driving heterogeneity in false positive rate. 
```

Comment:
Interesting, where did you learn that these specific biological factors can influence test results?